aten/src/ATen/core/interned_strings.h

```diff
@@ -28,6 +28,7 @@ namespace c10 {
   _(prim, Eval)                    \
   _(prim, Expand) /* onnx */       \
   _(prim, FusionGroup)             \
+  _(prim, TVMGroup)             \
```

Comment:
There is not need to add backend-specific entries to interned_strings.  `Symbol::fromQualString("tvm::TVMGroup")` will work fine. Code generators should not add ops to the `prim` namespace. That is reserved for core primitives.